project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject com.walmartlabs/schematic "1.1.0"
+(defproject com.walmartlabs/schematic "1.2.0-SNAPSHOT-1"
```

Comment:
What is this odd version number or?

Reply:
That was just so I could use the updated code in my other project (Clockwork) which is where I stumbled on this issue.  I can back that out and we can do a proper 1.2.0.